railties/test/application/rake/dbs_test.rb

```diff
@@ -484,19 +484,47 @@ def db_structure_dump_and_load(expected_database)
         end
       end
 
+      ["dump", "load"].each do |command|
+        test "db:structure:#{command} is deprecated" do
+          add_to_config("config.active_support.deprecation = :stderr")
+          stderr_output = capture(:stderr) { rails("db:structure:#{command}", stderr: true, allow_failure: true) }
+          assert_match(/DEPRECATION WARNING: Using `bin\/rails db:structure:#{command}` is deprecated and will be removed in Rails 6.2/, stderr_output)
+        end
+
+        test "db:structure:#{command}:NAME is deprecated" do
+          app_file "config/database.yml", <<-YAML
+            default: &default
+              adapter: sqlite3
+            development:
+              primary:
+                <<: *default
+              animals:
+                <<: *default
+                database: db/animals_development.sqlite3
+          YAML
```

Comment:
These tests should be in multi-db not dbs test since this is testing multiple databases.